### PR TITLE
Define scenario comparison view

### LINKED ISSUE #18: Descriptions of views are missing
- start new scenario
- load old scenario
- scenario comparison
- current scenario

### PR DESCRIPTION
**Pull request by TheOneAndra**
Fix #18 

**Changes proposed in this pull request**:
- Draft for new views

The following steps were realized, as well (if applies):
- [ ] Update the CHANGELOG.md

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl
-institut/open_plan/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

**Comment by TheOneAndra** [comment]
@Bachibouzouk I cloned the old "views" branch and reorganised this one for more clarity.

File: docs/views/Scenario_comparison.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.. _<view_name>-label: scenario_comparison
```
Comment (Bachibouzouk) [inline docs/views/Scenario_comparison.rst:1]: This needs to be placed where this file is included in tool_interface.rst

**Review by Bachibouzouk — APPROVED** [review]
You can update the CHANGELOG.md file to describe in one line each thing that you add/change/remove (one line per thing, minimum one line, can also be several lines spread witing add/remove/change categories)

You need to include the view in the RTD structure using `include`

Then you can merge your changes (you can always look at the steps in CONTRIBUTING.md file)

**Comment by Bachibouzouk** [comment]
meta-comment: the title of the PR can be more informative as "define new views". New is always relative, so avoid using it to describe changes. Make sure you capitalize the first letter

**Review by Bachibouzouk — APPROVED** [review]

**Pull request merged**